src/main/java/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -181,6 +183,7 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         searchView = (SearchView) MenuItemCompat.getActionView(item);
         searchView.setOnQueryTextListener(this);
         searchView.setOnCloseListener(this);
+        themeSearchView(getContext(), searchView, true);
```

Comment:
```suggestion
        ThemeUtils.themeSearchView(getContext(), searchView, true);
```

So we know that is a static call

Reply:
fixed